Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -425,6 +428,14 @@ private void InitializeServiceLocator()
             isInitializing = false;
         }
 
+        private void EnsureEditorSetup()
+        {
+            if (ActiveProfile.RenderDepthBuffer)
+            {
+                CameraCache.Main.gameObject.EnsureComponent<DepthBufferRenderer>();
```

Comment:
Is there a chance / downside to this script getting saved in the scene and existing at runtime?

Reply:
DepthBufferRenderer body of code is #IF UNITY_EDITOR so nothing will run in that case